### PR TITLE
Update Web-Interface.md

### PR DESCRIPTION
**Pull request by Petrprogs**

### CODE REVIEW

File: docs/Web-Interface.md [hunk 1]
```diff
@@ -85,6 +85,17 @@ If the web browser and the Transmission daemon are on the machine you can use ht
 | 5                 | Downloading speed                                     |
 | 6                 | Uploading speed                                       |
 
-
+### 4. List of added torrents ###
+![image](https://user-images.githubusercontent.com/76732207/181936421-a6d36ca0-e214-4f37-9368-1404cbeb125c.png)
```
Comment (nevack) [inline docs/Web-Interface.md:89]: nit: IMO it's better to have these images committed into repo.

Comment (Petrprogs) [reply docs/Web-Interface.md:89]: But i dowt know, where to put them. `docs/resources` ?

Comment (nevack) [reply docs/Web-Interface.md:89]: Yup, this path is ok.
You can do it for all image files listed in this markdown in a separate PR.

Comment (Petrprogs) [reply docs/Web-Interface.md:89]: Ok, thanks!

Comment (ckerr) [reply docs/Web-Interface.md:89]: This is a good idea. There are a few other GitHub image links in the docs too. I'd be :+1: on those being moved to `docs/resources/` as well

**Comment by Petrprogs** [comment]
Like to contributing to You!

**Pull request merged**